docs/pages/ds-display_L3_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,185 @@
+# Device Settings Display L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+- [Level 3 Python Test Cases High Level Overview](#level-3-python-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings Display module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL`  \- Hardware Abstraction Layer, may include some common components
+- `UT`   \- Unit Test(s)
+- `OEM`  \- Original Equipment Manufacture
+- `L3`   \- Level 3 Testing
+- `DS`   \- Device Settings
+- `EDID` \- Extended Display Identification Data
+- `DUT`  \- Device Under Test
+- `RAFT` \- Rapid Automation Framework for Testing
+- `Y`    \- yes supported
+- `NA`   \- Not Supported
+
+
+### References
+
+- `High Level Test Specification` - [dsDisplay High Level TestSpec](ds-display-high-Level_TestSpec.md)
+
+- `HAL Interface file` - [dsDisplay Header 4.0.0](https://github.com/rdkcentral/rdk-halif-device_settings/blob/4.0.0/include/dsDisplay.h)
+
+## Level 3 Test Cases High Level Overview
+
+Below are the top test use cases for the display.
+
+|#|Test-case|Description|Focus APIs|Source|Sink|
+|-|---------|-----------|----------|------|----|
+|1|Verify Display Events with callbacks|Monitor and verify that the callbacks are triggered correctly for each display event |`dsRegisterDisplayEventCallback()`|`Y`|`Y`|
+|2|Gets the EDID information|Verify the functionality of retrieving the EDID information of a display |`dsGetEDID()`|`Y`|`Y`|
```

Comment:
from 2-4 test case please mention how it will verified ?

Reply:
Done

Reply:
verification method is not clear. Something like:
"Connect various pre-defined displays to the device, retrieve their EDID information, and compare it against the expected EDID data"